AHBC_MIDTERM_2019_JULY_TEAMROCKET/ICash.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
+{
+    interface ICash
+    {
+        void PayWithCash(double total);
+
```

Comment:
Same here whatever we had in our notes from what we wanted the properties of cash to  be should be included in the interface